EPlast/EPlast.Tests/Controllers/UserControllerTests.cs

```diff
@@ -0,0 +1,610 @@
+            Assert.IsInstanceOf<NotFoundResult>(result);
+        }
+
+        [Test]
+        public async Task ApproveUser_ReturnsOkResult()
+        {
+            // Assert
+            var idString = "1";
+
+            _confirmedUserService
+                .Setup((x) => x.CreateAsync(It.IsAny<ClaimsPrincipal>(), idString, It.IsAny<bool>(), It.IsAny<bool>()));
+
+            // Act
+            var result = await _userController.ApproveUser(idString, It.IsAny<bool>(), It.IsAny<bool>());
+
+            // Assert
+            Assert.IsInstanceOf<OkResult>(result);
+        }
+
+        [Test]
+        public async Task ApproverDelete_NullUserIdString_ReturnsNotFoundResult()
+        {
+            // Act
+            var result = await _userController.ApproverDelete(It.IsAny<int>());
+
+            // Assert
+            _loggerService.Verify((x) => x.LogError(It.IsAny<string>()), Times.Once);
+            Assert.IsInstanceOf<NotFoundResult>(result);
+        }
+
+        [Test]
+        public async Task ApproverDelete_ReturnsOkResult()
+        {
+            // Assert
+            var confirmedId = 1;
+
+            _confirmedUserService
+                .Setup((x) => x.DeleteAsync(It.IsAny<int>()));
+
+            // Act
+            var result = await _userController.ApproverDelete(confirmedId);
+
+            // Assert
+            _loggerService.Verify((x) => x.LogInformation(It.IsAny<string>()), Times.Once);
+            Assert.IsInstanceOf<OkResult>(result);
+        }
+
+        public UserDTO CreateFakeUser()
+            => new UserDTO()
+            {
+                Id = "1",
+                FirstName = "SomeFirstName",
+                LastName = "SomeLastName",
+                UserProfile = new UserProfileDTO()
+                {
+                    EducationId = 1,
+                    WorkId = 1,
+                },
+            };
+
+        public UserViewModel CreateFakeUserViewModel()
+            => new UserViewModel()
+            {
+                ID = "1",
+            };
+
+        public EditUserViewModel CreateFakeEditUserViewModel()
+            => new EditUserViewModel()
+            {
+                User = new UserViewModel()
+                {
+                    ID = "1",
+                },
+
+                ImageBase64 = "SomeImgInBase64",
+                
+                EducationView = new UserEducationViewModel()
+                {
+                    PlaceOfStudyID = 1,
+                    SpecialityID = 1,
+                },
+                
+                WorkView = new UserWorkViewModel()
+                {
+                    PlaceOfWorkID = 1,
+                    PositionID = 1,
+                },
+            };
+
+        public UserInfoViewModel CreateFaceUserInfoViewModel()
+            => new UserInfoViewModel()
+            {
+                Id = "1",
+            };
+
+        public List<ConfirmedUserViewModel> CreateListOfConfirmedUserViewModels()
```

Comment:
If you have faking function, use private modification.
```suggestion
        private List<ConfirmedUserViewModel> CreateListOfConfirmedUserViewModels()
```

Reply:
Done.